app-frontend/src/app/pages/projects/edit/aoi-approve/aoi-approve.html

```diff
@@ -0,0 +1,32 @@
+<div class="sidebar-project">
+  <div class="sidebar-header">
+    <a class="btn sidebar-header-nav-btn" ui-sref="projects.edit">
+      <i class="icon-arrow-left"></i>
+    </a>
+    <h5 class="sidebar-title">AOI Scene Approval</h5>
```

Comment:
That's a good point -- I think we should defer to the mockups for now. There will be places in the UI where it isn't realistic to display the full text, but we should try and make it as clear as possible.